app/assets/javascripts/sitewide/form_validators.js.erb

```diff
@@ -142,11 +143,28 @@ function filesSizeValidator(ev, fileInputs, fileTypeEnum) {
     }
   };
 
+  function checkFilesTotalSize(fileInputs) {
+    if (!fileInputs || fileInputs < 2) {
+      return ;
+    }
+
+    var size = 0;
+    _.each(fileInputs, function(fileInput) {
+      var file = fileInput.files[0]
+      size += file.size;
+    })
+
+    if (size > fileTypeEnum) {
+      return "<%= I18n.t('general.file.total_size', size: Constants::FILE_MAX_SIZE_MB) %>".strToErrorFormat();
+    }
+  }
+
   // Check if any file exceeds allowed size limit
   var filesSizeValid = true;
   _.each(fileInputs, function(fileInput) {
     var file = fileInput.files[0];
     var assetError = getFileTooBigError(file);
+    var assetError = checkFilesTotalSize(fileInputs);
```

Comment:
So if I understand correctly, this is done for each file? Couldn't you just call this function (`checkFilesTotalSize(fileInputs)`) once outside of the loop, save its results into a variable, and then check that inside the loop?